<launch>

	<node pkg="turtlesim" type="turtlesim_node" name="simulator"/>

	<node pkg="turtlesim" type="turtle_teleop_key" name="keyboard"/>

	<node pkg="study_pkg" type="turtle_tf_broadcaster.py" name="turtle1_tf_broadcaster">
		<remap from="input_pose" to="turtle1/pose"/>
		<param name="turtle_tf_name" type="string" value="turtle1"/>
	</node>

	<node pkg="study_pkg" type="turtle_tf_broadcaster.py" name="turtle2_tf_broadcaster">
		<remap from="input_pose" to="turtle2/pose"/>
		<param name="turtle_tf_name" type="string" value="turtle2"/>
	</node>

	<node pkg="study_pkg" type="turtle_tf_listener.py" name="tf_turtle" />


</launch>